jetstream/sponsored-suggestion-placement-phase-2.toml

```
@@ -0,0 +1,658 @@
+)"""
+data_source = "urlbar_search_sessions"
+friendly_name = "SERP impressions per exposed search session"
+description = "Per-client rate of tagged Google SERP impressions per exposed urlbar search session"
+
+[metrics.exposed_serps_per_session.statistics.bootstrap_mean]
+[metrics.exposed_serps_per_session.statistics.deciles]
+
+[metrics.exposed_serps_with_ads_per_session]
+select_expression = """SAFE_DIVIDE(
+    COUNTIF(exposed_session AND serp_fields.has_ads_loaded),
+    COUNTIF(exposed_session AND new_session_start)
+)"""
+data_source = "urlbar_search_sessions"
+friendly_name = "SERP impressions with ads per exposed search session"
+description = "Per-client rate of tagged Google SERP impressions with ads loaded per exposed urlbar search session"
+
+[metrics.exposed_serps_with_ads_per_session.statistics.bootstrap_mean]
+[metrics.exposed_serps_with_ads_per_session.statistics.deciles]
+
+[metrics.exposed_ad_clicks_per_session]
+select_expression = """SAFE_DIVIDE(
+    COALESCE(SUM(IF(exposed_session, serp_fields.num_ad_clicks, 0)),0),
+    COUNTIF(exposed_session AND new_session_start)
+)"""
+data_source = "urlbar_search_sessions"
+friendly_name = "Ad clicks per exposed search session"
+description = "Per-client rate of SERP ad clicks per exposed urlbar search session"
+
+[metrics.exposed_ad_clicks_per_session.statistics.bootstrap_mean]
+[metrics.exposed_ad_clicks_per_session.statistics.deciles]
+
+[metrics.exposed_adm_impressions_per_session]
+select_expression = """SAFE_DIVIDE(
+    COUNTIF(exposed_session AND urlbar_fields.has_sponsored_impression),
+    COUNTIF(exposed_session AND new_session_start)
+)"""
+data_source = "urlbar_search_sessions"
+friendly_name = "Sponsored suggestion impressions per exposed search session"
+description = "Per-client rate of sponsored suggestion impressions per exposed urlbar search session"
+
+[metrics.exposed_adm_impressions_per_session.statistics.bootstrap_mean]
+[metrics.exposed_adm_impressions_per_session.statistics.deciles]
+
+[metrics.exposed_adm_clicks_per_session]
+select_expression = """SAFE_DIVIDE(
+    COUNTIF(
+        exposed_session
+        AND urlbar_fields.event_action = 'engaged'
+        AND urlbar_fields.product_selected_result = 'admarketplace_sponsored'
+    ),
+    COUNTIF(exposed_session AND new_session_start)
+)"""
+data_source = "urlbar_search_sessions"
+friendly_name = "Sponsored suggestion clicks per exposed search session"
+description = "Per-client rate of sponsored suggestion clicks per exposed urlbar search session"
+
+[metrics.exposed_adm_clicks_per_session.statistics.bootstrap_mean]
+[metrics.exposed_adm_clicks_per_session.statistics.deciles]
+
+[data_sources]
+
+[data_sources.tagged_google_serp_events]
+from_expression = """(
+    SELECT
+        * EXCEPT (is_ad_component),
+        component IN ('ad_carousel', 'ad_image_row', 'ad_link', 'ad_sidebar',
+        'ad_sitelink') AS ad_component
+    FROM `mozdata.firefox_desktop.serp_events`
```

```suggestion
    FROM `moz-fx-data-shared-prod.firefox_desktop_derived.serp_events_v1`
```
I suggest querying from the underlying v1 table just for this experiment as we're working on a v2, and this will ensure it'll use the expected table.